docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -51,3 +49,169 @@ described below, and relative to the joint TMREL.
++----------+---------------------------------------------------------+
+|  334     | H influenzae type B meningitis                          |
++----------+---------------------------------------------------------+
+|  335     | meningococcal meningitis                                |
++----------+---------------------------------------------------------+
+|  336     | other meningitis                                        |
++----------+---------------------------------------------------------+
+|  337     | encephalitis                                            |
++----------+---------------------------------------------------------+
+|  381     | neonatal preterm birth complications                    |
++----------+---------------------------------------------------------+
+|  382     | neonatal encephalopathy due to birth asphyxia and trauma|
++----------+---------------------------------------------------------+
+|  383     | Neonatal sepsis and other neonatal infections           |
++----------+---------------------------------------------------------+
+|  384     | hemolytic disease and other neonatal jaundice           |
++----------+---------------------------------------------------------+
+|  385     | other neonatal disorders                                |
++----------+---------------------------------------------------------+
+|  686     | sudden infant death syndrome                            |
++----------+---------------------------------------------------------+
+
+Restrictions
+++++++++++++
+
+Assumptions and limitations
++++++++++++++++++++++++++++
+
+Because the relative risks from GBD are for all cause mortality in the early and late neonatal period, we will apply the following equation to calculate all mortality in the early and late neonatal age groups
+
+The all cause mortality rate (ACMR) is comprised of the sum of: 
+
+   - causes that are unaffected by lbwsg 
+   - causes that are affected by lbwsg but are NOT modelled (due to some reason we cannot model them)
+   - causes that are affected by lbwsg AND modelled 
+
+ACMR =  :math:`\sum\limits_{\text{unaffected_causes}}\text{CSMR} + \sum\limits_{\text{affected_unmodelled_causes}}\text{CSMR} + \sum\limits_{\text{affected_modelled_causes}}\text{CSMR}`
+
+   |  i = low birth weight short gestation category
+   |  mr_i = mortality hazard in neonatal period for category i
+   |  RR_i = relative risk for all cause mortality in category i
+   |  state = either 1 with condition or 0 without condition 
+
+Hence, the mortality hazard for an individual in category i, mr_i, assuming that the relative risks for LBWSG applies only to causes that GBD considers to be affected by LBWSG. :
+
+:math:`\sum\limits_{\text{unaffected_causes}}\text{CSMR} + (ACMR - \sum\limits_{\text{unaffected_causes}}\text{CSMR} - \sum\limits_{\text{affected_modelled_causes}}\text{CSMR} + \sum\limits_{\text{modelled_causes}}\text{EMR_state} \times \text{(1-PAF_mostdetailed)} \times RR_i`
+
```

Comment:
this equation goes off of the page after you do "make html". Probably either need to figure out how to make this render on multiple lines or write it as plain text or a code block so that it can be easily seen when rendered.

Reply:
But the actual equation looks good to me!

Reply:
some of the equations from other docs are also going off the page in my browser...anyone have any idea how to fix this off the top of their heads? otherwise I can fiddle.